csharp/Adapter/Microsoft.Spark.CSharp/Configuration/ConfigurationService.cs

```diff
@@ -150,14 +150,25 @@ internal SparkCLRLocalConfiguration(System.Configuration.Configuration configura
 
             internal override string GetCSharpRDDExternalProcessName()
             {
-                return appSettings.Settings["CSharpWorkerPath"].Value;
+                KeyValueConfigurationElement workerPathConfig = appSettings.Settings["CSharpWorkerPath"];
+                if (workerPathConfig != null)
+                {
+                    // Use the file name part of the CSharpWorkerPath value, if specified
+                    return Path.GetFileName(workerPathConfig.Value);
```

Comment:
My question is prompted by what is the right value to be returned from GetCSharpRDDExternalProcessName.

Shouldn't *ProcessPath return the full path to the .exe and *ProcessName just the "file" part?
For example: `C:\my\path\CSharpWorker.exe` and `CSharpWorker.exe` respectively?

If that is the case, then can't *ProcessName automatically be derived using Path.GetFileName on the full path value returned from GetCSharpWorkerPath reading App.config setting?

I don't think I have quite grokked the difference between the expected config for "local" and "debug" execution modes.